### ISSUE TITLE
Error installling sharp v0.16.2 (via grunt-sharp)

### ISSUE THREAD

**Issue by anonimoconiglio** [lovell/sharp#2175]
Sorry for my noobness :( I really tried everything (removing node_modules, package-lock.js, and ~/node-gyp folder) this error does not go away and I'm out of ideas here. 

**The problem**: whenever I tried to install grunt-sharp with `npm install` it returns me an error: 
```shell
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:194:23)
gyp ERR! stack     at ChildProcess.emit (events.js:315:20)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:275:12)
gyp ERR! System Linux 5.6.5-arch3-1
gyp ERR! command "/usr/bin/node" "/usr/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /home/santiago/Desktop/SANDBOX/new-grunt-image/node_modules/sharp
gyp ERR! node -v v13.13.0
gyp ERR! node-gyp -v v5.1.0
gyp ERR! not ok 

[...]

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! sharp@0.16.2 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the sharp@0.16.2 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

Do you guys have any idea? I tried to install an old version of node locally (like node@10 or node@11), then it installs grunt-sharp but when I launch the task it returns me a different error, something like "sharp was compiled against a different Node.js version"

**enviroment**: 
**npm** 6.14.4 
**node** v13.13.0
**node-gyp** v6.1.0

**Comment by lovell** [comment]
Hi, please can you upgrade the version of sharp; the latest is v0.25.2.

sharp v0.16.2 (2016) is too old to work with actively-supported versions of Node.js 10+

**Comment by anonimoconiglio** [comment]
Hi @lovell thanks for the answer. You helped me to solve the problem!
 
I gathered that the problem came from `grunt-sharp` (since I'm not installing sharp directly, I'm using grunt-sharp package as specified in the title of the issue).

So I took a look at the repository and found out that it's not quite updated (last update was 4 years ago), but someone forked the project and specified `"sharp": "latest"` on the package.json of the module. So i set that repository manually on my devDepencencies 
```json
  "devDependencies": {
    "grunt": "^1.1.0",
    "grunt-sharp": "https://github.com/verlico/grunt-sharp/archive/master.tar.gz",
    "otherpackages":"etc..."
}
```
And now it installs and works! 
Hope this could be useful for those who trip over the same problem
